add LTS policy (#3228)
AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet: README.md

> **Why IdentityModel?**
> - **Widely Adopted:** Trusted by thousands of developers to integrate OIDC and OAuth2.0 standards.
> - **Secure by Design:** Built with security as a priority to reduce common vulnerabilities.
> - **Extensible:** Easily extend or customize for advanced use cases.
> - **Battle hardened:** Validates 5+ trillion requests daily, and growing.

## Versions

You can find the release notes for each version [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases). Older versions can be found [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/Release-Notes).

## Version Lifecycle and Support Matrix

See [Long Term Support policy](./supportPolicy.md) for details.

| Major Version | Last Release | Status |
| --------------|--------------|--------|
| 8.x           | 8.9.0        | Active (Current) |
| 7.x           | 7.7.1        | Supported (LTS) through .NET 8 LTS lifetime Nov 10, 2026|
| 6.x           | 6.36.0       | Not supported since May 2024|

> | 5.x           | 5.7.0        | Supported Microsoft.Owin.Security.JWT 4.2.2 |

Reviewer: This is not clear to me, is 5.x supported or tied to the lifetime of Microsoft.Owin.Security.JWT 4.2.2?

Reviewer: ```suggestion
| 5.x           | 5.7.0        | Supported (LTS), tied to the Microsoft.Owin.Security.JWT 4.2.2 lifetime |
```

Author: We'd like to not support it, but it's only supported because it's in Owin. :(

## IdentityModel 8.x?

Version `8.x` introduces significant updates and improvements:
- **Enhanced Performance:** Optimized token validation to handle high-throughput scenarios.
- **.NET Compatibility:** Fully compatible with .NET 9.

>🧭LTS: Supported through .NET 9 LTS lifetime: May 12, 2026 + .NET 10 LTS (~3 years).

> ## IdentityModel 7x

Reviewer: ```suggestion
## IdentityModel 7.x
```

> [IdentityModel 7x](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/IdentityModel-7x) introduced several improvements related to serialization and consistency in the API, which will provide a better user experience for developers, as well as full AOT compatibility on .NET, and huge perf improvements compared to 6x.

Reviewer: ```suggestion
[IdentityModel 7x](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/IdentityModel-7x) introduced several improvements related to serialization and consistency in the API, which provide a better user experience for developers, as well as full AOT compatibility on .NET, and considerable performance improvements compared to IdentityModel 6x.
```

>🧭LTS: Supported through .NET 8 LTS lifetime: Nov 10, 2026.
>

> >⚡Recommendation: Move to 8x.

Reviewer: ```suggestion
>⚡Recommendation: Move to 8.x.
```

## IdentityModel 6.x

>🧭Deprecated: Support ended with .NET 7 LTS lifetime: May 2024.
>

> >⚡Action: Move to 8x.

Reviewer: ```suggestion
>⚡Action: Move to 8.x.
```

## IdentityModel 5.x

__Not a recommended version__

>🧭LTS: Supported for Microsoft.Owin.Security.JWT
>

> >⚡Action: Move to 8x.

Reviewer: ```suggestion
>⚡Action: Move to 8.x.
```
